config.json

```diff
@@ -1,7 +1,8 @@
 {
   "develop": {
     "11155111": {
-      "salt": 0
+      "salt": 0,
+      "owner": "0x921d360aD22A6D0289ce57fcb8250e299cB19EA3"
```

Comment:
no need for this change

Reply:
Forge project check pipeline failed without that line ATM

Reply:
forge project check pipeline failed without that line

Reply:
i just commented out this liine for now in the config function as it was purely just a helper method